pkg/config/constants/constants.go

```diff
@@ -125,4 +125,7 @@ const (
 	LocalPathStorageNamespace string = "local-path-storage"
 
 	TestVMLabel = "istio.io/test-vm"
+
+	// TrustworthyJWTPath is the defaut 3P token to authenticate with third party services
```

Comment:
I am not sure istiod is a "third party services"

Reply:
The Third Party token is not always used to communicate just with Istiod - It can also be used to contact other Cloud services as well right?

Reply:
yes. Maybe it should be "istiod or third part services"